extra/resources/ifspeed

```diff
@@ -7,12 +7,16 @@
 # Copyright (c) 2011 Vladislav Bogdanov <bubble@hoster-ok.com>
 # Partially based on 'ping' RA by Andrew Beekhof
 #
+# Change on 2017 by Tomer Azran <tomerazran@gmail.com>:
+#     Add "ip" parameter to detect network interface name by ip address:
+#     http://lists.clusterlabs.org/pipermail/users/2017-August/006224.html
+#
 # OCF instance parameters:
 #    OCF_RESKEY_name:         name of attribute to set in CIB
 #    OCF_RESKEY_iface:        network interface to monitor
 #    OCF_RESKEY_bridge_ports: if not null and OCF_RESKEY_iface is a bridge, list of
 #                             bridge ports to consider.
-#                             Default is all ports which have designated_bridge=root_id 
+#                             Default is all ports which have designated_bridge=root_id
```

Comment:
This is fine, but for future reference, we discourage formatting-only changes (even when it's an improvement). It distracts from the real changes when reviewing, and there's always a chance of an unintentional mistake.